ops/jenkins/bfd-deploy-apps/Jenkinsfile

```diff
@@ -5,6 +5,7 @@
  * This script will be run by Jenkins when deploying apps and app AMIs.
  * </p>
  */
+@Library('bfd@bfd-2647-allow-regression-suite-ephemeral')
```

Comment:
So is this needed permanently to make things build? Seems weird to have the branch name in there, if so.

Reply:
Ugh, nope! I didn't realize my last commit hadn't pushed to remove this. Removed!